assets/scss/_chunk.scss

```diff
@@ -0,0 +1,16 @@
+/*doc
+---
+title: Chunk
+---
+Adds bottom space to an element
+or a group of elements.
+```
+*/
+%chunk,
+.chunk {
+	@extend %clearfix;
+	display: block;
+	@include responsiveBase() {
+		margin-bottom: $base!important; // why is this being set with important in sassquatch?
```

Comment:
This is `!important` in SQ2 because we ran into some issues combining `chunk` with other classes. Because this is basically just a style override anyway, let's just leave this `!important` for now